templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -19,17 +19,18 @@
     <jackson.version>$[jackson.version]</jackson.version> <!-- Overriding Jackson for fixing vulnerabilities -->
     <guava.version>$[guava.version]</guava.version>
     <devonfw.test.excluded.groups>system</devonfw.test.excluded.groups>
+    <app.version>${revision}${changelist}</app.version>
   </properties>
 
   <modules>
     <module>api</module>
     <module>core</module>
-#if ($earProjectName != '.')
+    #if ($earProjectName != '.')
     <module>${earProjectName}</module>
-#end
-#if ($batch == 'batch')
+    #end
+    #if ($batch == 'batch')
     <module>batch</module>
-#end
+    #end
```

Comment:
Seems Eclipse formatter also indents velocity macros.
We need to check the generated output as this might break the indentation there.
I hoped eclipse would leave things like this as is...
If this is a problem in the resulting pom we can fix in the new story for improvement.